journalpump/senders/kafka.py

```diff
@@ -110,9 +110,14 @@ def send_messages(self, *, messages, cursor):
         if not self.kafka_producer:
             self._init_kafka()
         try:
-            for msg in messages:
-                self.kafka_producer.send(topic=self.topic, value=msg, key=self.kafka_msg_key)
+            list_fut_record_metadata = [
+                self.kafka_producer.send(topic=self.topic, value=msg, key=self.kafka_msg_key) for msg in messages
+            ]
             self.kafka_producer.flush()
+            for fut_record_metadata in list_fut_record_metadata:
```

Comment:
or just `record_metadata`?

Reply:
naming the loop variable `result_future` and the list `result_futures`